package.json

```diff
@@ -22,7 +22,7 @@
   "homepage": "https://github.com/seek-oss/browserslist-config-seek#readme",
   "devDependencies": {
     "@commitlint/cli": "^6.0.5",
-    "browserslist": "3.0.0",
+    "browserslist": "3.1.0",
```

Comment:
@michaeltaranto seems like the version pinning isn't working 🤔

Reply:
It is working just we evergreen `browserslist` too on this package in `renovate.json`

Reply:
What I mean is, I would have thought this config would mean `browserslist` is set to the latest major version: https://github.com/seek-oss/renovate-config-seek/blob/3b4c52a3e9f6e6eba9fb76388f346b042c25bfce/package.json#L46-L48

Reply:
Probably need a longer discussion about how this is working. I think the "fix" for what you're looking for is to update `browserslist` to `^3.0.0`. As without the range we are asking Renovate to keep it up to date, regardless of semver.